oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollectorIT.java

```diff
@@ -2659,8 +2652,6 @@ public void testGCDeletedPropsWithDryRunMode() throws Exception {
 
     @Test
     public void testDeletedPropsAndUnmergedBCWithCollisionWithDryRunMode() throws Exception {
-        // OAK-10869:
-        assumeTrue(fullGcMode != FullGCMode.ORPHANS_EMPTYPROPS_KEEP_ONE_ALL_PROPS);
```

Comment:
This was introduced as a result of the mentioned flaky failure on mongo. Thus same question: why re-add it without any underlying test fix?

Reply:
I can close this PR and create a new one after further investigation

Reply:
+1, might make sense to get the `assertEventually` improvement in and split this one here to a new PR

Reply:
I reverted deleted assumeTrue statements and keep assertEventually changes.